src/MooLite/core/inventory/items/EquippedItems.ts

```diff
@@ -0,0 +1,9 @@
+class EquippedItems {
+    name;
+    level;
+    constructor(name, level) {
+        this.name = name;
+        this.level = level;
+    }
+}
+export default EquippedItems;
```

Comment:
Why do we need a specific interface for this? Is this not just a `CharacterItem`